app/navigation/navigationTheme.js

```diff
@@ -0,0 +1,11 @@
+import { DefaultTheme } from "@react-navigation/native";
+import colors from "../views/styles/colors";
+
+export default {
+  ...DefaultTheme,
```

Comment:
I am curious to know, why this is required? Will this be needed for all type of theme files?

Reply:
For our app there would be only one Navigation Theme file, unless in future we support in-app user configured themes. We if are defining our own navigation theme for customizing colors of text in navigation footer then its the best practice to first spread out the default theme from react-navigation/native and then override only the part that we want.